Makefile

```diff
@@ -26,6 +26,7 @@ install:
 	cp -R config/* $(DESTDIR)$(sysconfdir)/$(package_name)
 	rm $(DESTDIR)$(sysconfdir)/$(package_name)/logrotate
 	install -Dpm0644 config/logrotate $(DESTDIR)$(sysconfdir)/logrotate.d/$(package_name)
+	rm $(DESTDIR)$(sysconfdir)/$(package_name)/osrt-slsa-user.conf
```

Comment:
Can be merged with the rm above

Reply:
Fixed: created a slsa directory and move related config files there